### ISSUE TITLE
Is there a callback I can hook into?

### ISSUE THREAD

**Issue by Quirksmode** [alexmingoia/express-snapshot#1]
Hi, 

Loving this package, solves a problem we were having. Once thing I need to do is run a script once the templates have generated. Is there a callback I can hook into?

Thanks

**Comment by alexmingoia** [comment]
No, you'd have to hook into the promise:  https://github.com/alexmingoia/express-snapshot/blob/master/index.js#L57
